StreamChatClient_v3/WebSocketClient/Events/Event.swift

```diff
@@ -15,11 +26,16 @@ protocol EventWithPayload: Event {
 }
 
 /// A protocol for user `Event`s.
-protocol UserEvent: Event {
+protocol EventWithUserPayload: EventWithPayload {
```

Comment:
What's wrong with `UserEvent`?

Reply:
I unified event protocols to make them a little bit more clear for combinations of event payloads.